Makefile

```diff
@@ -45,13 +45,11 @@ generate-standalone:
 generate-type-helpers:
 	$(NODE) packages/babel-types/scripts/generateTypeHelpers.js
 
-build-typings: build-flow-typings build-typescript-typings
-
 build-flow-typings:
 	$(NODE) packages/babel-types/scripts/generators/flow.js > packages/babel-types/lib/index.js.flow
 
-build-typescript-typings:
-	echo "TODO: maybe add types for typescript 3.7"
+build-typescript-3.7-typings:
+	$(NODE) packages/babel-types/scripts/generators/typescript-3.7.js > packages/babel-types/lib/index-ts3.7.d.ts
```

Comment:
would like to not have duplicated script for typescript type generation… - going to try https://github.com/sandersn/downlevel-dts

Reply:
I'd prefer to keep this solution that has been working well for a while. We'll remove it in Babel 8.